source/_integrations/onkyo.markdown

```diff
@@ -37,6 +34,8 @@ Input sources:
   type: list
 {% endconfiguration_basic %}
 
+The above can also be adjusted later using the **Reconfigure** flow.
```

Comment:
I've already had multiple people ask me how to change it, and even make GitHub issues about it. This is in fact the improvement that I most care about, as it should hopefully decrease my support burden.

Reply:
The problem is: Reconfigure flow, is a developer thing. Nobody knows what this means.

Reply:
I'm open to different wording on that, if you prefer something else.

It is called `Reconfigure` in the UI, so that should be OK, I think, but you are right that `flow` is a developer term.

Reply:
I adjusted the phrasing, borrowing it from the `ring` integration docs.